www-apps/radicale/radicale-2.1.8.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python{3_4,3_5,3_6} )
+
+inherit eutils distutils-r1 user
+
+MY_PN="Radicale"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="A simple CalDAV calendar server"
+HOMEPAGE="http://radicale.org/"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+bcrypt"
+
+RDEPEND=">=dev-python/vobject-0.9.5[${PYTHON_USEDEP}]
+	bcrypt? ( dev-python/passlib[bcrypt,${PYTHON_USEDEP}] )"
+
+S=${WORKDIR}/${MY_P}
+
+RDIR=/var/lib/radicale
+LDIR=/var/log/radicale
+
+pkg_setup() {
+	enewgroup radicale
+	enewuser radicale -1 -1 ${RDIR} radicale
+}
+
+python_install_all() {
+	rm README* || die
+
+	# init file
+	newinitd "${FILESDIR}"/radicale.init.d radicale
+
+	# directories
+	diropts -m0750
+	dodir ${RDIR}
+	fowners radicale:radicale ${RDIR}
+	keepdir ${RDIR}
+	diropts -m0755
+	dodir ${LDIR}
+	fowners radicale:radicale ${LDIR}
+	keepdir ${LDIR}
+
+	# config file
+	insinto /etc/${PN}
+	doins config logging
+
+	# fcgi and wsgi files
+	exeinto /usr/share/${PN}
+	doexe radicale.wsgi
+	doexe radicale.fcgi
+
```

Comment:
This could be done under the global USE-flag "cgi"

Reply:
Sorry, two questions about this. First, do you mean only the `.fcgi` file or the `.wsgi` file as well? Second, I thought this was in violation of the “don’t use USE flags to control installation of small files that don’t affect dependencies” rule (I don’t remember where that rule is documented, but I remember hearing something about it on the mailing list in the past)?

Reply:
The files first looked like they would only be needed when running radicale under an external webserver, maybe that is not even true and the builtin server uses them as well. So not sure whether they can/should be left out. The rule you suggest sounds sensible, just leave it like that.